### PR TITLE
Motion study: MPMLink and AI integration

### PR DESCRIPTION
**Pull request by hobinjk-ptc**
This PR got a bit out of hand and should be like 3-4 separate PRs made over the course of the past month+ so....enjoy!

### CODE REVIEW

File: src/humanPose/LensProvider.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import {RebaLens} from "./RebaLens.js";
+import {OverallRebaLens} from "./OverallRebaLens.js";
+import {MuriLens} from "./MuriLens.js";
+import {ValueAddWasteTimeLens} from "./ValueAddWasteTimeLens.js";
+import {AccelerationLens} from "./AccelerationLens.js";
+import {PoseObjectIdLens} from "./PoseObjectIdLens.js";
+
+export class LensProvider {
```
Comment (dangond-ptc) [inline src/humanPose/LensProvider.js:8]: Nice cleanup!

File: src/motionStudy/config.js.template [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const windchillBaseUrl = 'https://<windchill instance';
```
Comment (dangond-ptc) [inline src/motionStudy/config.js.template:1]: Missing a `>`

File: src/motionStudy/index.js [hunk 3]
```diff
@@ -143,8 +184,15 @@ import {MotionStudyMobile} from './MotionStudyMobile.js'
             motionStudyByFrame[msgData.frame].hydrateMotionStudy(msgData.analyticsData);
         });
 
+        realityEditor.network.addPostMessageHandler('analyticsSetSensor', (msgData) => {
+            console.log('set sensor', msgData);
```
Comment (dangond-ptc) [inline src/motionStudy/index.js:188]: Probably don't want to commit these logging messages?

**Review by dangond-ptc — APPROVED** [review]

File: src/motionStudy/index.js (hunk 3 above)
Comment (hobinjk-ptc) [reply src/motionStudy/index.js]: Thanks! Cleaned up throughout

**Pull request merged**